go.mod

```diff
@@ -23,7 +23,6 @@ require (
 )
 
 replace (
-	github.com/gardener/machine-controller-manager => github.com/gardener/machine-controller-manager v0.34.3
```

Comment:
I had misunderstood your suggestion earlier. I have updated it now. Thanks for the suggestion. Updated the go version while at it.